packages/documentation/copy/pt/reference/Namespaces and Modules.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Namespaces e Módulos
+layout: docs
+permalink: /docs/handbook/namespaces-and-modules.html
```

Comment:
To fix the CI error you need to change the permalink to `/pt/docs/handbook/namespaces-and-modules.html`.